user_documentation_cf_cli_plugin.md

```diff
@@ -24,6 +26,9 @@ Subhankar Chattopadhyay
    1. [Listing all backups of a service-instance](#listing-all-backups-of-a-service-instance)
    1. [Starting a restore](#starting-a-restore)
    1. [Aborting a restore](#aborting-a-restore)
+   1. [Listing backups for deleted service instances](#listing-deleted-backups)
+   1. [Listing backups for deleted service instances using instance guid](#listing-deleted-backups-guid)
```

Comment:
Just to keep inline with above,
"Listing all backups of a service-instance by instance guid"